src/rules.c

```diff
@@ -537,10 +537,10 @@ void SchemaPrefixes_RemoveSpec(StrongRef ref) {
   IndexSpec *spec = StrongRef_Get(ref);
   if (!spec || !spec->rule || !spec->rule->prefixes) return;
 
-  const char **prefixes = spec->rule->prefixes;
+  sds *prefixes = spec->rule->prefixes;
   for (int i = 0; i < array_len(prefixes); ++i) {
```

Comment:
In the future PR, where `arr_free_ex` is fixed, `arr_len` and other functions should also be fixed to accept `const void*` (`const_arry_t`?).